tests/unit/test_datasets_downloader.py

```diff
@@ -1,10 +1,13 @@
-import asyncio
```

Comment:
Would you mind keeping [imports ordered as suggested in PEP8](https://www.python.org/dev/peps/pep-0008/#imports)? First native modules, then third-party and so on… you can use tools as `isort` or `prospector` to help you out ; )